compiler/rustc_hir/src/intravisit.rs

```diff
@@ -31,7 +31,7 @@
 //!      impl into scope while visiting the impl-items, and then back out again.
 //!    - How: Implement `intravisit::Visitor` and override the `NestedFilter` type to
 //!      `nested_filter::All` (and implement `maybe_tcx`). Walk your crate with
-//!      `tcx.hir().walk_toplevel_module(visitor)` invoked on `tcx.hir().krate()`.
```

Comment:
Nit/Preexisting: After #136466, `tcx.hir().krate()` no longer exists, though I'm not sure whether this comment makes sense even if that becomes `tcx.hir_crate(())`.

Reply:
Hmm, I don't even know what the "invoked on" part is supposed to mean. `walk_toplevel_module` was invoked on `Map`, is now invoked on `TyCtxt`, there's no `Crate` value directly involved. I have just removed that final part of that sentence.